Src/FluentAssertions/Formatting/FormattingOptions.cs

```diff
@@ -31,12 +38,47 @@ public class FormattingOptions
     /// Because of technical reasons, the actual output may be one or two lines longer.
     /// </para>
     /// <para>
-    /// This value should not changed on <see cref="AssertionOptions.FormattingOptions"/> from within a unit test.
+    /// This value should not be changed on <see cref="AssertionOptions.FormattingOptions"/> from within a unit test.
     /// See the <see href="https://fluentassertions.com/extensibility/#thread-safety">docs</see> on how to safely use it.
     /// </para>
     /// </remarks>
     public int MaxLines { get; set; } = 100;
 
+    /// <summary>
+    /// Removes a scoped formatter that was previously added though <see cref="AddFormatter"/>.
```

Comment:
```suggestion
    /// Removes a scoped formatter that was previously added through <see cref="FormattingOptions.AddFormatter"/>.
```

Also at previous location on `Formatters`